### PR TITLE
[gui] Show line numbers in the report list view

### PR DESCRIPTION
**Pull request by csordasmarton**

### CODE REVIEW

File: web/server/vue-cli/src/views/Reports.vue [hunk 1]
```diff
@@ -115,7 +115,7 @@
             }}"
             class="file-name"
           >
-            {{ item.checkedFile }}
+            {{ item.checkedFile }} @ Line {{ item.line }}
```
Comment (bruntib) [inline web/server/vue-cli/src/views/Reports.vue:118]:
```suggestion
            {{ item.checkedFile }} @&nbsp;Line&nbsp;{{ item.line }}
```
Otherwise the line is broken at weird places. By the way what is the motivation behind this change? We should consider a more concise format:
```suggestion
            {{ item.checkedFile }}:{{ item.line }}
```

Comment (csordasmarton) [reply web/server/vue-cli/src/views/Reports.vue]: Good point to not to brake the line at weird places. By the way this is how it looked like in the old UI.
